challenges/faq_accordion_card/index.html

```diff
@@ -26,9 +26,127 @@
+          alt=""
+        />
+        <div class="relative hidden w-1/2 overflow-hidden lg:block">
+          <img
+            class="absolute -left-24"
+            src="images/bg-pattern-desktop.svg"
+            alt=""
+          />
+          <img
+            class="-ml-16"
+            src="images/illustration-woman-online-desktop.svg"
+            alt=""
+          />
+        </div>
+
+        <div class="relative mx-auto mb-2 block w-1/2 lg:hidden">
+          <img
+            class="-mt-24"
+            src="images/illustration-woman-online-mobile.svg"
+            alt=""
+          />
+          <img
+            class="absolute top-0"
+            src="images/bg-pattern-mobile.svg"
+            alt=""
+          />
+        </div>
+
+        <div class="mt-5 p-4 lg:mt-0 lg:w-1/2 lg:p-0">
+          <div class="lg:w-4/5">
+            <h1
+              class="text-very-dark-desaturated-blue text-center text-4xl font-bold lg:text-left"
+            >
+              FAQ
+            </h1>
+            <div
+              class="w-full py-6"
+              x-data="{
+                faqs: 
+                [
+                  {
+                    question: 'How many team members can I invite?',
+                    answer: 'You can invite up to 2 additional users on the Free plan. There is no limit on team members for the Premium plan.',
+                    isOpen: true
+                  },
+                  {
+                    question: 'What is the maximum file upload size?',
+                    answer: 'No more than 2GB. All files in your account must fit your allotted storage space.',
+                    isOpen: false
+                  },
+                  {
+                    question: 'How do I reset my password?',
+                    answer: 'Click “Forgot password” from the login page or “Change password” from your profile page. A reset link will be emailed to you.',
+                    isOpen: false
+                  },
+                  {
+                    question: 'Can I cancel my subscription?',
+                    answer: 'Yes! Send us a message and we’ll process your request no questions asked.',
+                    isOpen: false
+                  },
+                  {
+                    question: 'Do you provide additional support?',
+                    answer: 'Chat and email support is available 24/7. Phone lines are open during normal business hours.',
+                    isOpen: false
+                  },
+
+                ],
+
+                baseQuestionClass: 'text-sm lg:text-base'
+
+              }"
+            >
+              <template x-for="faq in faqs">
+                <div
+                  class="flex cursor-pointer items-start justify-between border-b py-3"
+                  @click="faqs = faqs.map(i => ({...i, isOpen: faq.question !== i.question ? false : !i.isOpen }))"
+                >
+                  <div class="w-11/12">
+                    <h5
```

Comment:
These need to be links (`<a>` tags) for the tests to pass.
Check out the tests - when you see the query "=What is the maximum file upload size?", it means link `<a>` tag that contains `What is the maximum file size?`.